dev/system-test/firestore.ts

```diff
@@ -173,6 +173,16 @@ describe('Firestore class', () => {
       unsubscribe();
     }
   });
+
+  it('throws an error if terminate() is called with pending BulkWriter operations', async () => {
+    const writer = firestore._bulkWriter();
+    const ref = randomCol.doc('doc-1');
+    writer.set(ref, {foo: 'bar'});
+    expect(firestore.terminate()).to.eventually.be.rejectedWith(
```

Comment:
This expectation is now a Promise that has to be awaited.

Reply:
done.